heudiconv/heuristics/reproin.py

```diff
@@ -124,6 +124,9 @@
 
 - We replace all ( with { and ) with } to be able e.g. to specify session {date}
 - "WIP " prefix unconditionally added by the scanner is stripped
+
+
+-Michael-Sun Plans on adding a heuristic to deal with ses-X* instances
```

Comment:
make it happen!

Reply:
add above , where `_ses-` marker described, description that `XXX` markers are dealt in some other way.